include/rms/rms.hh

```diff
@@ -116,32 +116,32 @@ template <std::size_t num_samples> class RMS {
     }
 
     /**
-    *   Join the calculation thread when the destructor is called.
-    */
+     *   Join the calculation thread when the destructor is called.
+     */
     ~RMS() {
         if (thread_alive) {
             thread.join();
         }
     }
 
     /**
-    * Explicitly delete the copy constructor
-    */ 
+     * Explicitly delete the copy constructor
```

Comment:
Why are all these changes to RMS here, also shouldn't we be avoiding destructors and copy constructors etc entirely now?

Reply:
fixed